Cargo.toml

```diff
@@ -17,3 +17,8 @@ exclude = [".gitignore", ".github/**"]
 maintenance = { status = "passively-maintained" }
 
 [dependencies]
+
+[features]
+default = ["std"]
+std = []
+alloc = []
```

Comment:
Please ensure that files always end with a newline character

Reply:
ok